src-docs/src/views/form_layouts/described_form_group.js

```diff
@@ -122,6 +122,7 @@ export default class extends Component {
                 { value: 'option_two', text: 'Option two' },
                 { value: 'option_three', text: 'Option three' },
               ]}
+              aria-label="euiSelect-help"
```

Comment:
For example, a good value here could be:
```suggestion
              aria-label="An example of a form element without a visible label"
```